src/mui/list/DatagridHeaderCell.js

```diff
@@ -36,52 +24,47 @@ export const DatagridHeaderCell = ({
     );
     return (
         <TableHeaderColumn style={style}>
-            {field.props.sortable !== false && field.props.source ? (
-                <FlatButton
-                    labelPosition="before"
-                    onClick={updateSort}
-                    data-sort={field.props.source}
-                    label={
-                      <span style={styles.nonSortableLabel}>
-                          {
-                              <FieldTitle
-                                  label={field.props.label}
-                                  source={field.props.source}
-                                  resource={resource}
-                              />
-                          }
-                      </span>
-                    }
-                    icon={
-                        field.props.source === currentSort.field ? (
-                            <ContentSort
-                                style={
-                                    currentSort.order === 'ASC' ? (
-                                        { transform: 'rotate(180deg)',
-                                          opacity:.3,
-                                        }
-                                    ) : (
-                                        {opacity:.3}
-                                    )
-                                }
-                            />
-                        ) : (
-                            false
-                        )
-                    }
-                    style={styles.sortButton}
-                />
-            ) : (
-                <span style={styles.nonSortableLabel}>
-                    {
+            <FlatButton
```

Comment:
Don't this make a button that doesn't work in some cases?

Reply:
yes, in some cases this button doesn't do anything, but the styles will always be the same. It is difficult otherwise to have the same style for when it's sortable & nonSortable.